packages/loot-core/src/server/main.js

```diff
@@ -1167,21 +1201,28 @@ handlers['account-unlink'] = mutator(async function ({ id }) {
     [bankId],
   );
 
-  if (count === 0) {
-    // No more accounts are associated with this bank. We can remove
-    // it from Plaid.
-
-    let [[, userId], [, key]] = await asyncStorage.multiGet([
-      'user-id',
-      'user-key',
-    ]);
+  // No more accounts are associated with this bank. We can remove
+  // it from Nordigen.
+  // TODO Check the flag
+  // if (local['flags.syncAccount']) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏do we need to check the feature flag here?

IMO if the flag is turned OFF (bank sync disabled), then `count` would always be `0`. So we never evaluate this expression.

Reply:
We should check it here but I don't know how to read the flag in this place... 
We need it because the `account-unlink` function is executed when the user closes the account. It'll fail in the following scenario - The user turned ON the flag and create a linked account. Later he switches the flag to OFF and he wants to close the account. Here the `count` returns more or equal to 1 and calls to nordigen endpoint.

Reply:
I think it’s fine to say that if you want to uncheck the feature flag you need to unlink all synced accounts or else you’ll run into issues.